cypress/integration/dimensions/period.cy.js

```diff
@@ -487,4 +465,77 @@ describe('Period dimension', () => {
             selectPeriodType(fixedPeriodTypes[1]) // Click the second item to close the dropdown
         })
     })
+    describe('using period settings - hidden bi-weekly', () => {
+        it('navigates to the start page', () => {
+            cy.intercept(
+                /systemSettings/,
+                systemSettingsInterceptFn({
+                    ...systemSettingsBody,
+                    keyHideBiWeeklyPeriods: true,
+                })
+            )
+            goToStartPage()
+        })
+        it('opens the period dimension modal', () => {
```

Comment:
Cypress testing best practices would suggest that all this should be in a single test:
https://docs.cypress.io/app/core-concepts/best-practices#Having-Tests-Rely-On-The-State-Of-Previous-Tests
https://docs.cypress.io/app/core-concepts/best-practices#Creating-Tiny-Tests-With-A-Single-Assertion

Reply:
Refactored.